site/blogs/roadToPerformance.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: blog
+title: The road to performance
+description: Lightning 3.0, our obsession with performance
+date: 2024-10-28
+author: Wouter-lucas van Boesschoten
+---
+
+# Blits: Unlocking Performance in Your Big Screen Experience
+
+**Performance-obsessed technology for contained rendering.**
+
+Blits is a brand new (well, 1 year old at the time of writing) JavaScript Application Development UI library for Lightning 3. Where Lightning 2 had a built-in application development syntax, Lightning 3 is split into two main building blocks: the renderer and Blits. This separation provides more flexibility and iterability between the two foundational pillars behind Lightning 3.
+
+The Smart TV, game console, and set-top box space is a niche within a niche, and Lightning is the key player in that space. Most platforms come with a form of browser—usually far older than the one you’re using to read this. The one feature that ties them together is support for WebGL, which makes rendering stable across a wide array of devices, models, and years—dating from TVs running browsers built in 2014 to more recent versions.
```

Comment:
browser-usually and years-dating seem a bit off

Reply:
yeah it started with stuff between brackets, then between -'s to emphasize a small contextual change in the narrative but the sentence in between got too big :D 

thanks! will fix